cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -5202,15 +5202,6 @@
     </description>
   </property>
 
-  <property>
-    <name>task.worker.preload.artifacts</name>
```

Comment:
renamed the config as artifact localizer is used across multiple worker types

Reply:
wondering if we need to have separate preload artifacts for each artifact localizer. I.e., pre-load different artifacts for task-worker and preview-runner.

Reply:
IIUC for validation and preview runs we need all artifacts for a pipeline so there is no difference between task-worker and preview-worker in the artifact requirement. Is there a usecase you are thinking of where taskworker needs different artifacts than preview runners ?

Reply:
I cannot think of any. SGTM.